src/components/Api.js

```diff
@@ -0,0 +1,108 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    // constructor body
+    this._baseUrl = baseUrl;
+    this._authorization = headers.authorization;
+  }
+
+  _checkServerResponse(res) {
```

Comment:
Great that you've implemented a  method for processing server response, so the code that checks the server response and converts it from JSON  is reused

Reply:
Thank you!